[New Concept & Exercise]: Iteration (#627)
exercism/crystal: exercises/concept/spellbound-steel/.docs/hints.md

# Hints

## 1. Find card in the deck

> - There is a [built-in method][index] that can find the first element in the collection that meets the condition specified in the block, and return its index or `nil` if no element is found.

Reviewer: ```suggestion
- There is a [built-in method][index] that can find the first element in the collection that meets the condition specified in the block and returns its index or `nil` if no element is found.
```

## 2. Make all characters capitalized

> - There is  a [built-in method][map] to transform all elements in the collection.

Reviewer: ```suggestion
- There is a [built-in method][map] to transform all elements in the collection.
```

> - There is a [built-in method][capitalize] to capitalize a words for `String`.

Reviewer: ```suggestion
- There is a [built-in method][capitalize] to capitalize all the words in a `String`.
```
